demo/src/app/app.module.ts

```diff
@@ -37,6 +38,7 @@ import { CaseViewConsumerComponent } from './case-view-consumer.component';
     CaseCreateConsumerComponent,
     CaseProgressConsumerComponent,
     CaseViewConsumerComponent,
+    CaseCreateFilterComponent
```

Comment:
This is part of a future task in this epic: https://tools.hmcts.net/jira/browse/RDM-3918

Reply:
Done